src/runner/provider/qti.js

```diff
@@ -267,7 +267,12 @@ var qtiItemRuntimeProvider = {
                 done(renderingQueue);
             }, this.getLoadedClasses());
         });
-    }
+    },
+
+    getApipData: function() {
+        var apipData = this._item && _.isFunction(this._item.getApipAccessibility) && this._item.getApipAccessibility() || null;
```

Comment:
Where is `getApipAccessibility` defined ? 
Do you need some parsing  or can you rely on the compiled item ?

Reply:
It is [here](https://github.com/oat-sa/tao-item-runner-qti-fe/blob/56e76ca96615167ab322e517a24f8c9f31df2d16/src/qtiItem/core/Item.js#L159)  in [Item.js](https://github.com/oat-sa/tao-item-runner-qti-fe/search?q=getApipAccessibility&unscoped_q=getApipAccessibility):
`getApipAccessibility: function getApipAccessibility() {
    return this.apipAccessibility;
},`